Makefile

```diff
@@ -80,6 +80,9 @@ test:		wake.db
 unittest:	all
 	$(WAKE_ENV) ./bin/wake --in test_wake runUnitTests
 
+remoteCacheTests:	all
+	$(WAKE_ENV) ./bin/wake -d -x 'testPostgres Unit'
```

Comment:
nit: Do we need `-d` here? Also should `testPostgres` be in `test_wake` like the other tests?

Reply:
-d It makes it a million times

Also I'm not going to mess with `test_wake` stuff right now since thats its own bag of worms